### ISSUE TITLE
CIL_FRAMEWORK_TESTS_2,5,6,8,9,11 fail in docker test on Travis

### ISSUE THREAD

**Issue by KrisThielemans** [SyneRBI/SIRF-SuperBuild#506]
https://travis-ci.com/github/SyneRBI/SIRF-SuperBuild/jobs/492098570#L13907-L13921
```
6: ERROR: test_algorithms (unittest.loader._FailedTest)
6: ----------------------------------------------------------------------
6: ImportError: Failed to import test module: test_algorithms
6: Traceback (most recent call last):
6:   File "/opt/pyvenv/lib/python3.8/unittest/loader.py", line 436, in _find_test_path
6:     module = self._get_module_from_name(name)
6:   File "/opt/pyvenv/lib/python3.8/unittest/loader.py", line 377, in _get_module_from_name
6:     __import__(name)
6:   File "/opt/SIRF-SuperBuild/sources/CCPi-Framework/Wrappers/Python/test/test_algorithms.py", line 47, in <module>
6:     from cil.utilities import dataexample
6:   File "/opt/SIRF-SuperBuild/INSTALL/python/cil/utilities/dataexample.py", line 29, in <module>
6:     from cil.io import NEXUSDataReader
6:   File "/opt/SIRF-SuperBuild/INSTALL/python/cil/io/__init__.py", line 25, in <module>
6:     from .TXRMDataReader import TXRMDataReader
6:   File "/opt/SIRF-SuperBuild/INSTALL/python/cil/io/TXRMDataReader.py", line 22, in <module>
6:     import olefile
6: ModuleNotFoundError: No module named 'olefile'
```
I guess we don't run these on normal Travis builds. In any case, we're missing `olefile`. Unfortunately, this makes all `docker` builds fail and stops us uploading anything on dockerhub

**Comment by casperdcl** [comment]
in many ways duplicate of the ol' #92 and #493?

**Comment by paskino** [comment]
Isn't `olefile` [installed](https://github.com/SyneRBI/SIRF-SuperBuild/blob/master/.travis.yml#L56) on travis already?

**Comment by paskino** [comment]
That's for docker. 
Anyway closed by #508
